.github/workflows/pr-audit.yml

```diff
@@ -16,17 +16,8 @@ permissions:
   contents: read
 
 jobs:
-  security-audit:
+  cargo-deny:
```

Comment:
If we are doing the same thing with both files, I think it would be better to merge them.

Reply:
The [docs](https://github.com/EmbarkStudios/cargo-deny-action) recommend using this:
```toml
name: CI
on: [push, pull_request]
jobs:
  cargo-deny:
    runs-on: ubuntu-22.04
    strategy:
      matrix:
        checks:
          - advisories
          - bans licenses sources

    # Prevent sudden announcement of a new advisory from failing ci:
    continue-on-error: ${{ matrix.checks == 'advisories' }}

    steps:
    - uses: actions/checkout@v3
    - uses: EmbarkStudios/cargo-deny-action@v1
      with:
        command: check ${{ matrix.checks }}
```
Perhaps we should do this in the toml for master, but keep it as-is on PRs?

Reply:
> Perhaps we should do this in the toml for master, but keep it as-is on PRs?

Hmm. This will cause the advisory to be ignored until someone next opens a PR that edits Cargo.toml, right?

Reply:
I guess that depends on whether we get an email on error ...

Reply:
I have an image that continue-on-error treats failures the same as successes, but in practice I have rarely used it.

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error

Reply:
Well, this PR is sufficient as-is to address the problem of broken CI, so I think it is fine to leave the fine tuning to future PRs.